app/grandchallenge/core/templatetags/process_tags.py

```diff
@@ -0,0 +1,25 @@
+from django import template
+from django.template.loader import render_to_string
+
+from grandchallenge.core.utils.tag_substitutions import TagSubstitution
+
+register = template.Library()
+
+process_youtube_tags = TagSubstitution(
+    tag_name="youtube",
+    replacement=lambda _id: render_to_string(
+        "partials/youtube_embed.html",
```

Comment:
The partial should be moved to a namespace:

```suggestion
        "grandchallenge/partials/youtube_embed.html",
```